genai-perf/tests/test_input_embeddings.py

```diff
@@ -55,9 +59,9 @@ def test_get_input_dataset_from_embeddings_file(self, mock_file, mock_exists):
             batch_size=batch_size,
             num_prompts=100,
         )
-        inputs = Inputs(config)
+        factory = InputRetrieverFactory(config)
```

Comment:
Could you use a different variable name? Factory is too generic.

Reply:
I have often seen short tests use generic names.
Ill update this and avoid that practice in the future.

Updated to `input_retriever_factory`

Reply:
Thanks! Could you do this for all of the tests? I think test_input_retriever_factory is the last one.

I think people used to prioritize conciseness more, but readability is always important, especially if you might use the variable more than 1-2 lines below its declaration. However, with code completion tools now widely available, the emphasis is shifting (at least in my opinion) to be readability >>> conciseness.

Reply:
done.

I agree with you. 
I didn't think this was really unreadable but if you are saying it was harder to follow, that is justification enough to change. Thanks for calling it out.